ldap/servers/slapd/back-ldbm/back-ldbm.h

```diff
@@ -573,12 +573,20 @@ struct ldbminfo
     char *li_backend_implement;          /* low layer backend implementation */
     int li_noparentcheck;                /* check if parent exists on add */
 
```

Comment:
My understanding is that dblock monitoring will remain specific to bdb. Would it make sense to define those fields in bdb_config rather than in generic ldbminfo struct ?

Reply:
Maybe I miss something but I need to check these variables in the ldbm_search.c - **li_dblock_monitoring, li_dblock_monitoring_pause, li_dblock_threshold_reached.** 

All is left is **li_dblock_threshold** so I've just left it here for consistency.

I see that we do the same for other bdb settings like **li_dblock** and others.

Reply:
That is right, some fields that are specific to bdb_config are still in ldbminfo. The move could be easy for some of them (like li_dblock or li_dbcachesize I guess) but it may be more complex for others like li_directory).
Now as we are introducing new fields we can put them directly in bdb_config

Reply:
As discussed on IRC and agreed, we won't move the fields to bdb_config as it may make code more obscure and fragile (as bdb_config is a thing in itself and it requires deeper integration). As the result, we'll need much more testing and it will delay the quality.
I've mentioned the additional information in the comments in the code as discussed on IRC.